extensions/amp-ad/0.1/amp-ad.css

```diff
@@ -67,9 +67,12 @@ amp-embed iframe {
   border-radius: 5px;
 }
 
-amp-ad[type=adsense] > iframe {
+amp-ad[data-a4a-upgrade-type="amp-ad-network-doubleclick-impl"] > iframe,
```

Comment:
We can't apply the min-width/min-height, since we don't programmatically adjust the iframe style on DF adsense, so I split this in two. Let me know what you think.